src/gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -8444,7 +8486,6 @@ properties:
       - Gynandroblastoma
       - Haemangioblastoma
       - Haemangiosarcoma
-      - Hairy cell leukaemia variant
```

Comment:
Please, confirm that there is no data in the graph currently using this value.

Reply:
Confirmed! 👍